gemonlinedb/.gitignore

```diff
@@ -3,6 +3,9 @@
 
 # Don't ignore common xsd definitions
 !xml/schema/common.xsd
+!xml/schema/system-topology.xsd
+!xml/examples/*.xml
+!xml/*.xml
```

Comment:
Any reason to not ignore all `.xml`/`.xsd` files in these locations?
```
!xml/**/*.xsd
!xml/**/*.xml
```

Reply:
`xsd` because some of them are generated. Could ignore all `xml` indeed.